common/interfaces/flash_interface/flash_api.h

```diff
@@ -297,6 +297,20 @@ int update_challenge_flash(const char *name,
                            const uint8_t target[SHA256_SIZE],
                            const uint8_t random_number[POW_RAND_NUMBER_SIZE]);
 
+/**
+ * The function sets the wallet locked status and updates the locked card
+ * number(1-4).
+ *
+ * @param wallet_id The wallet_id is used to determine which wallet is locked.
+ * If that wallet is found, it's status is updated to locked
+ * @param card_locked It is used to determine on which card is the wallet
+ * locked. The value of "card_locked" should be between 1 and 4, inclusive.
+ *
+ * @return SUCCESS if execution is successful, error is returned if any error is
+ * encountered duirng wallet fetching.
+ */
+int set_wallet_locked(const char *wallet_name, uint8_t card_locked);
```

Comment:
the 2nd param name does not fit for readability. Suggestion:

`card_locked` -> `card_number`

Reply:
updated